ciao-controller/api/api.go

```diff
@@ -736,5 +790,15 @@ func Routes(config Config) *mux.Router {
 	route.Methods("PUT")
 	route.HeadersRegexp("Content-Type", matchContent)
 
+	// configuration
+	matchContent = fmt.Sprintf("application/(%s|json)", ClusterConfigV1)
+
+	route = r.Handle("/configuration", Handler{context, showConfig, true})
+	route.Methods("GET")
+	route.HeadersRegexp("Content-Type", matchContent)
+
+	route = r.Handle("/configuration", Handler{context, updateConfig, true})
+	route.Methods("POST")
+	route.HeadersRegexp("Content-Type", matchContent)
```

Comment:
I am thinking that the verb for this should probably be a PUT since you aren't creating new configuration parameters, just updating.